app/Container.php

```diff
@@ -4,7 +4,9 @@
 
 use Nette;
 
-class Container extends Nette\Object {
+class Container {
+
+	use Nette\SmartObject;
```

Comment:
Tohle je nevhodné, protože jsi to udělal nekompatibilní se starým nette. Nevím jestli je to pro vás problém.